src/core/qgsmaplayer.h

```diff
@@ -1420,7 +1420,12 @@ class CORE_EXPORT QgsMapLayer : public QObject
     void opacityChanged( double opacity );
 
     /**
-     * Signal emitted when renderer is changed.
+     * Emitted when the layer's style is changed. This includes any changes which affect the layer's
```

Comment:
hum, you added that comment to the doc of rendererChanged(), not styleChanged()

Reply:
Ooops....

Reply:
Fixed